delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -797,13 +650,12 @@ def getDistributionConfig(distributionID) {
 	def distributionConfig
 	try {
 		distributionConfig = sh (
-			script: "aws cloudfront get-distribution-config \
+			script: "aws cloudfront get-distribution-service_config \
 						--output json --id "+distributionID,
 			returnStdout: true
 		)
 		return distributionConfig
 	}catch (ex) {
-		updateServiceInDB(var_db_service_id, configLoader.AWS.API.PROD_ID, configLoader.AWS.REGION, "deletion_failed");
 		send_status_email ("FAILED")
```

Comment:
send an event for failure

Reply:
Modified the code for sending failed event